### PR TITLE
[Cherry Pick] Node preview state bar addition

### PR DESCRIPTION
**Pull request by Amoursol**
### Purpose

Due to concerns around low contrast of the existing Preview On/Off state, particularly at zoomed out levels, we've moved to using a grey-bar at the base of nodes that sits inside of the same space as the Errors/Warnings/Info bar, which serves a similar zoom purpose.

The preview status bar is in conjunction with the eye icon on the nodes and intended to be the first step in assisting users in a zoomed out graph state. Future additions of nodal zoom thresholds will improve upon this.

Hierarchy of Status Bar is as follows: Error > Warning > Info > Preview.

This means the Preview bar state will not show if any of the other three states are in play. Note that the eye icon will still show in all cases where the node preview is turned off.

All nodes below have been tested, including state changes, UI nodes, input nodes and standard nodes. 

image

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Release Notes

Addition of grey node preview status bar at base of node when preview turned off. 

### Reviewers

@DynamoDS/dynamo 

### FYIs

@mjkkirschner
@OliverEGreen

### CODE REVIEW

**Comment by Amoursol** [comment]
Outstanding questions/issues to resolve:
1. Code Block Nodes don't behave correctly. It appears they are not updated upon copy/paste so may be missing a trigger. Any thoughts @QilongTang or @mjkkirschner? 

https://user-images.githubusercontent.com/12857357/144457310-bccbb287-db7f-46ea-8d31-b965a9103a4c.mov

2. The name of the status bar in code is called "NodeWarningBar", which should probably be retooled into a generic status bar. Thoughts?

File: src/DynamoCoreWpf/ViewModels/Core/PortViewModel.cs [hunk 1]
```diff
@@ -574,7 +574,7 @@ private void OnMouseLeftUseLevel(object parameter)
         /// </summary>
         protected virtual void RefreshPortColors()
         {
-            PortBackgroundColor = node.IsVisible ? PortBackgroundColorDefault : PortBackgroundColorPreviewOff;
+            PortBackgroundColor = PortBackgroundColorDefault;
```
Comment (mjkkirschner) [inline src/DynamoCoreWpf/ViewModels/Core/PortViewModel.cs:577]: if the background is always being set to the same color just set it once.

Comment (Amoursol) [reply src/DynamoCoreWpf/ViewModels/Core/PortViewModel.cs:577]: Discussed in person with @mjkkirschner. Leaving for now.

File: src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs [hunk 2]
```diff
@@ -362,7 +362,7 @@ public bool IsVisible
         /// node has undismissed info/warning/error messages.
         /// </summary>
         [JsonIgnore]
-        public bool NodeWarningBarVisible => ErrorBubble != null && ErrorBubble.DoesNodeDisplayMessages;
+        public bool NodeWarningBarVisible => ErrorBubble != null && ErrorBubble.DoesNodeDisplayMessages || IsVisible == false;
```
[the remark this answers is not in the document]
Comment (Amoursol) [reply src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs:365]: Discussed in person with @mjkkirschner. Leaving for now.

File: src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs [hunk 3]
```diff
@@ -1,67 +0,0 @@
-﻿using System;
```
Comment (QilongTang) [inline src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs:1]: Please revert the change, you are deleting this file as part of this PR

Comment (QilongTang) [reply src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs:1]: In the future, if you accidentally included change to this file, you can commit what the number was before, then it will appear you have not touched this file in PR

**Review by QilongTang — CHANGES_REQUESTED** [review]
Please revert deleting `src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs`

**Comment by Amoursol** [comment]
> Please revert deleting `src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs`

Should be resolved now @QilongTang

**Comment by QilongTang** [comment]
Regression: `Dynamo.Tests.GraphLayoutTests.GraphLayoutComplex`

**Comment by QilongTang** [comment]
@Amoursol @mjkkirschner The self serve job passed. Merging. 

@Amoursol Please also cherry-pick if you can. Let me know
